tools/targets/__init__.py

```diff
@@ -418,8 +418,8 @@ def is_PSA_non_secure_target(self):
         return 'NSPE_Target' in self.labels
 
     @property
-    def is_PSA_target(self):
-        return self.is_PSA_secure_target or self.is_PSA_non_secure_target
+    def is_TFM_target(self):
+        return getattr(self, 'tfm_target_name', False)
```

Comment:
Did single-core PSA targets use `is_PSA_target()` at all?

Reply:
@Patater No, it was used to identify Armv8-M and dual core PSA targets.

Reply:
OK